pdks/Gf180/gf180_hdl21/pdk_data.py

```diff
@@ -62,34 +62,50 @@ class MosParams:
     w = h.Param(dtype=h.Scalar, desc="Width in PDK Units (µm)", default=1 * µ)
     l = h.Param(dtype=h.Scalar, desc="Length in PDK Units (µm)", default=1 * µ)
     nf = h.Param(dtype=h.Scalar, desc="Number of Fingers", default=1)
-    # This unfortunate naming is to prevent conflicts with base python.
-    As = h.Param(
+
+    #! CAUTION: The subsequent parameters are not recommended for design use.
+
+    ad = h.Param(
         dtype=h.Scalar,
-        desc="Source Area",
-        default=h.Literal("int((nf+2)/2) * w/nf * 0.18u"),
+        desc="Drain Area",
+        # `0.18u` is diffusion length (m)
+        # As `nf` increments, drain region quantity is alternatively -1 or == source region quantity
+        default=h.Literal("int((nf+1)/2) * w/nf * 0.18u"),
     )
 
-    ad = h.Param(
+    # This unfortunate naming is to prevent conflicts with base python.
+    As = h.Param(
```

Comment:
Yeah that's a shame.  

Does Python have any kinda escape hatch for this?  
In Rust you add a little `r#` "literal syntax naming" prefix to names like this (e.g. `r#as` or `r#import`). I don't know of any such Python-ism.

Reply:
AFAIK, it's a reserved keyword and there's no way around it.